manifests/01-tsb-catalogsource-configmap.yaml

```diff
@@ -0,0 +1,262 @@
+                verbs:
+                - create
+              - apiGroups:
+                - template.openshift.io
+                resources:
+                - brokertemplateinstances
+                verbs:
+                - create
+                - delete
+                - get
+                - update
+              - apiGroups:
+                - template.openshift.io
+                resources:
+                - brokertemplateinstances/finalizers
+                verbs:
+                - update
+              - apiGroups:
+                - template.openshift.io
+                resources:
+                - templateinstances
+                verbs:
+                - assign
+                - create
+                - delete
+                - get
+              - apiGroups:
+                - template.openshift.io
+                resources:
+                - templates
+                verbs:
+                - get
+                - list
+                - watch
+              - apiGroups:
+                - ""
+                resources:
+                - secrets
+                verbs:
+                - create
+                - delete
+                - get
+              - apiGroups:
+                - ""
+                resources:
+                - configmaps
+                - services
+                verbs:
+                - get
+              - apiGroups:
+                - ""
+                resources:
+                - routes
+                verbs:
+                - get
+              - apiGroups:
+                - route.openshift.io
+                resources:
+                - routes
+                verbs:
+                - get
+              - apiGroups:
+                - ""
+                resources:
+                - events
+                verbs:
+                - create
+                - patch
+                - update
+            deployments:
+            - name: template-service-broker-operator
+              spec:
+                replicas: 1
+                selector:
+                  matchLabels:
+                    name: template-service-broker-operator-alm-owned
+                template:
+                  metadata:
+                    name: template-service-broker-operator-alm-owned
+                    labels:
+                      name: template-service-broker-operator-alm-owned
+                  spec:
+                    serviceAccountName: template-service-broker-operator
+                    containers:
+                    - name: template-service-broker-operator
+                      image: docker.io/automationbroker/template-service-broker-operator:latest
+                      imagePullPolicy: IfNotPresent
+                      env:
+                      - name: IMAGE
+                        value: docker.io/openshift/origin-template-service-broker:latest
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
```

Comment:
I think if we make this `value: openshift-template-service-broker` then we would be able to create the TSB operator where they want it (`rh-operators`) and just deploy the broker where it belongs.